broker/service/api/__init__.py

```diff
@@ -42,6 +42,12 @@
         if plugin != '' and plugin not in config.sections():
             raise Exception("plugin '%s' section missing" % plugin)
 
+
+    if 'persistence' in config.sections():
+        persistence_name = config.get('persistence', 'persistence_name')
```

Comment:
I think there should be a default persistance, so we don't rely on configuration by default.

Reply:
In PR #52 this issue is being resolved.